appengine-deploy/action.yml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: Deploy to Google App Engine
+author: GoogleCloudPlatform
+description: |-
+  App Engine is a fully managed, serverless platform for developing and hosting
```

Comment:
Consider tailoring the action's description to the functionality provided by the action itself. Perhaps linking to App Engine documentation for a description of the service.

Reply:
done.